tests/Logging.XUnit.Tests/Integration/DatabaseFixture.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Martin Costello, 2018. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace MartinCostello.Logging.XUnit.Integration
+{
+    public sealed class DatabaseFixture : IAsyncLifetime
+    {
+        private readonly ILogger _loggerInitialize;
+        private readonly ILogger _loggerDispose;
```

Comment:
Took me a second to understand the naming here. Maybe swap the word order?

Reply:
Addressed in 095a649903f023288e925ddc53e62076efaf4c60. Two loggers are not actually needed but that was a *trick* to get code coverage on `Xunit.Abstractions.IMessageSinkExtensions.ToLogger<T>`.